src/dd-win-prof/dd-win-prof.vcxproj

```diff
@@ -71,14 +72,34 @@
       <EnableUAC>false</EnableUAC>
       <AdditionalDependencies>dbghelp.lib;ws2_32.lib;userenv.lib;ntdll.lib;crypt32.lib;secur32.lib;bcrypt.lib;psapi.lib;powrprof.lib;ncrypt.lib;windowsapp.lib;$(ProjectDir)..\..\third-party\libdatadog-x64-windows\$(Configuration)\static\datadog_profiling_ffi.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
+    <PreBuildEvent>
+      <Command>copy dd-win-prof.h ..\reference</Command>
```

Comment:
will this be OK ? Or is it ..\..\

Reply:
the .h file needs to be copied in the `src\reference` folder so `..\reference` from the `src\dd-win-prof` folder